### DISCUSSION TITLE
Check a page has children?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6032] [General]
How would I perform a check if a page has children?

![Screenshot 2022-05-12 at 10 13 47](https://user-images.githubusercontent.com/72254741/168036797-df00fc2a-5377-411d-b436-91c736a5e54b.png)

I thought this would work, but it doesn't

```
{{ if page:children }}
    YUP
{{ /if }}
 ```
 
 ```
Statamic 3.3.11 Pro
Laravel 9.5.1
PHP 8.0.8
simonhamp/statamic-heroicons 1.0.1
statamic/collaboration 0.4.0

parser: runtime
```

**Comment by stephenmeehanuk** [comment]
Looks like this is a known issue: https://github.com/statamic/cms/issues/5567

**Reply by ryanmitchell** [reply]
Did you give `{{ if (page:children | count) > 0 }} {{ /if }}` a go?

**Reply by stephenmeehanuk** [reply]
Thanks for the reply, I did try that - unfortunately it doesn't work. I think it's a bug, I found two other people trying to do the same thing.

**Reply by edalzell** [reply]
what does `{{ page:children | dump }}` do?

**Reply by stephenmeehanuk** [reply]
Apologies for the late reply, `{{ page:children | dump }}` does nothing, not output. {{ page:id | dump }} works. Others have the same issue https://github.com/statamic/cms/issues/5567

**Reply by edalzell** [reply]
ya the children things is only if you're in a structure, which I presume you're not.

**Reply by stephenmeehanuk** [reply]
It is a structure though? I think I've got the same issue as this https://github.com/statamic/cms/issues/5567

I'm trying to display the `children` on the `corporate responsibility` page.

<img width="914" alt="Screenshot 2022-05-19 at 10 10 03" src="https://user-images.githubusercontent.com/72254741/169257667-b910db7a-2d8f-4047-883e-90c97638a73e.png">

**Reply by edalzell** [reply]
gotta be inside a `nav` tag then, that's a structure
